### DISCUSSION TITLE
bulgaria

### DISCUSSION THREAD

**Discussion by Meno12ivanov** [microsoft/vscode-cpptools#13031] [Q&A]
[variants-schema.json](https://github.com/user-attachments/files/18050141/variants-schema.json)

**Comment by Meno12ivanov** [comment]
> [variants-schema.json](https://github.com/user-attachments/files/18050141/variants-schema.json)

**Reply by Meno12ivanov** [reply]
> > [variants-schema.json](https://github.com/user-attachments/files/18050141/variants-schema.json)

**Comment by bobbrow** [comment]
That file is from CMake Tools. If you have a question about it, please ask it in that extension's repository: https://github.com/microsoft/vscode-cmake-tools
